west.yml

```diff
@@ -27,6 +27,8 @@ manifest:
       url-base: https://github.com/zephyrproject-rtos
     - name: civetweb
       url-base: https://github.com/antmicro
+    - name: juullabs
+      url-base: https://github.com/JuulLabs-OSS
```

Comment:
We might want to discuss having a fork of it, although in this case it's perhaps not worth it.